composer.json

```diff
@@ -16,10 +16,12 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
-        "ircmaxell/password-compat": "~1.0",
-        "paragonie/random_compat": "~1.0|~2.0|~9.99",
```

Comment:
password-compat and password-compat could be adde to the replace section: `"...compat": "*"` (same in the PHP7 polyfill)

Reply:
I have added them to the replace section, as well as the `ircmaxell/password-compat` to php 5.5